stable/chartmuseum/README.md

```diff
@@ -219,12 +219,68 @@ env:
     STORAGE_GOOGLE_PREFIX:    
 ```
 
+### Using with Google Cloud Storage and a Google Service Account
+
+A Google service account credentials are stored in a json file. There are two approaches here. Ideally you don't want to send your secrets to tiller. In that case, before installing this chart, you should create a secret with those credentials:
+
+```shell
+kubectl create secret generic chartmuseum-secret --from-file=credentials.json="my-project-45e35d85a593.json"
+```
+
+Then you can either use a `VALUES` yaml with your values or set those values in the command line:
+
+```shell
+helm install stable/chartmuseum --debug  --set gcp.secret.enabled=true,env.open.STORAGE=google,env.open.DISABLE_API=false,env.open.STORAGE_GOOGLE_BUCKET=my-gcp-chartmuseum,gcp.secret.name=chartmuseum-secret
+```
+
+If you prefer to use a yaml file:
+
+```yaml
+env:
+  open:
+    STORAGE: google
+    STORAGE_GOOGLE_BUCKET: my-gcs-bucket
+    STORAGE_GOOGLE_PREFIX:
+    GOOGLE_SERVICE_ACCOUNT: true
+existing:
+  secret:
+    gcp:
+      enabled: true
+      secretName: chartmuseum-secret
+      key: credentials.json
+```
```

Comment:
With the latest changes, this example is obsolete (references to `env.open.GOOGLE_SERVICE_ACCOUNT`) and `existing.secret.gcp.secreName`

Reply:
I've seen them, I was fixing them 😄

Reply:
I believe it is gcp.secret.`name`, not `secretName`. ;-)

Reply:
eeeeeek 😂